app/smartapi.py

```diff
@@ -62,9 +64,16 @@ def get_trapi_endpoints(self):
             except (KeyError, IndexError):
                 source_url = None
             try:
-                version = hit["info"]["x-trapi"]["version"]
-            except KeyError:
                 version = None
+                regex = re.compile("[0-9].[0-9].")
+                trapi_minor = regex.match(self.trapi).group()
+                # check the TRAPI version against workflow-runner TRAPI version
+                if hit["info"]["x-trapi"]["version"].startswith(trapi_minor):
```

Comment:
If we specify a full version in this service, then that would cause us to miss any patch versions. Maybe we just never specify a full version?

Reply:
That should handle only matching the major and minor version, but not the patch version. So any patch version will be added, but only the exact major and minor.

Reply:
My bad. regex at the end of the work day is hard.